packages/template-block/src/components/TemplatePreview.tsx

```diff
@@ -60,7 +63,10 @@ export const TemplatePreview = ({
                         borderRadius,
                         border,
                         height,
+                        cursor: enableCta ? 'pointer' : 'default',
                     }}
```

Comment:
Maybe better to add `disabled` to the button, or is this not suitable, then you also don't need to handle the onClick...

Reply:
Good catch; that's actually a leftover of when I was trying to use a div - which of course doesn't make semantical sense. It's fixed and simplified in 4c6ce2602243c786d0ed66d081f0040cb43c0aa9